src/attributes.ts

```diff
@@ -14,7 +14,7 @@ export class Attributes {
     cur1stStave: number,
     duration: number,
     notes: Array<VF.Note>,
-    systems: VF.System[]
+    systems: System[]
```

Comment:
Why is System no longer VF.System and Note stays as VF.Note?

Reply:
VF.System needs to be updated to include the additional types. Without this, there will be type errors.